README.md

```diff
@@ -37,6 +37,7 @@ MAX_QUERY_NUMBER=<optional, integer number, maximum number of records to be retu
 MAX_QUERY_ROUNDS=<optional, integer number, maximum number of round trips to database done by a single query for records>
 MAX_QUERY_TIME_MILSECS=<optional, integer number, maximum number of time for database query>
 FEE_RATE=<optional, decimal number, use to calculate minimal l2tx fee required, default to 0>
+ENABLE_DEBUG_RPC=<optional, boolean, default to false>
```

Comment:
It mainly focuses on profiling. What about renaming all "debug" to "profile".

Reply:
will update rpc method to `prof_cpu` and `prof_memory`, env var to `ENABLE_PROF_RPC `

Reply:
updated